vignettes/blueprintr.Rmd

```diff
@@ -40,88 +40,113 @@ cache_location <- tempdir()
+`blueprintr` is a framework for managing your data assets in a reproducible fashion. While it uses [drake](https://github.com/ropensci/drake) or [targets](https://cran.r-project.org/web/packages/targets/), it adds automated steps for tabular dataset documentation and testing. This allows researchers to create a replicable framework to prevent programming issues from affecting analysis results.
 
-# Basic Use
+## Installation 
 
-The first, and recommended, step is to attach blueprintr to your R session with `library()`.
+```{r setup, results= FALSE}
+# install.packages("remotes")
+# remotes::install_github("nyuglobalties/blueprintr")
 
-```{r setup}
 library(blueprintr)
 ```
 
-In a [drake project](https://books.ropensci.org/drake/projects.html), all packages that you want attached are declared in a `"packages.R"` file. This `library(blueprintr)` command should go there.
+## Designed Use of blueprintr
+`blueprintr` provides your data with guardrails typically found in software engineering workflows. 
+This allows you to test and document before deploying to production.
 
-blueprintr is built around "blueprints." Our first blueprint will be a blueprint for `mtcars`:
+The top level of the `blueprintr` workflow is a "blueprints" directory, consisting of `.R` files-- each a blueprint. 
 
-```{r}
-blueprint(
-  "mtcars_dat",
-  description = "The famous mtcars dataset",
-  command = {
-    mtcars
-  }
-)
-```
+### About blueprints
+Each blueprint has two components to it: 
+* Data Construction Spec, usually a `.R` file that instructs drake or targets on how to build a specific dataset. 
+* Metadata, usually a `.csv` file that incorporates any mapping files and checks that need to be done on the dataset. 
 
-All blueprints have
+In order to create a blueprint, we use the `blueprint` function. This function takes three arguments: name (the name of your generated dataset), description (a description of your dataset), command (any functions that need to be applied in order to build the dataset). 
 
-* A name (the first argument) for the _target_ dataset.
-* A description or brief summary of what the target is. Can be `NULL`.
-* A command, which is a quoted statement that has the code for building this target.
-* A metadata location, which is a path to where the target metadata is saved.
+A project may need only a few blueprints, but more likely you'll need nested blueprints to transform the data. 
 
-<div class="vg-warning"><span>the blueprint name is "mtcars_dat" rather than "mtcars". If the two had the same name, drake would determine that the blueprint has a _circular dependency_ (it depends on itself). To avoid this, blueprints should not have the same names as global variables, like `mtcars`.</span></div>
+`blueprintr` has six "steps" (targets) per blueprint:
```

Comment:
Maybe say "generates" instead of "has" to indicate what exactly blueprintr does?